clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonCachingAgentFilterConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package com.netflix.spinnaker.clouddriver.aws.provider.agent;
+
+import java.util.List;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties("aws.filter")
```

Comment:
Let's move this to `aws.caching.filter` and we'll be good to go.

Reply:
Sure thing - done.